doc/source/configuration/ci-cd.rst

```diff
@@ -122,21 +127,32 @@ Workflow Deployment
 
 2. Run :code:`kayobe playbook run ${KAYOBE_CONFIG_PATH}/ansible/write-github-workflows.yml`
 
-3. Add all required secrets to repository either via the GitHub UI or GitHub CLI (may require repository owner)
+3. Add all required secrets and variables to repository either via the GitHub UI or GitHub CLI (may require repository owner)
```

Comment:
Please make separate lists of variables and secrets so that users know which to set.

Reply:
Addded table to represent this.